packages/kbn-cell-actions/src/components/inline_actions.tsx

```diff
@@ -18,18 +19,22 @@ interface InlineActionsProps {
   actionContext: CellActionExecutionContext;
   showActionTooltips: boolean;
   visibleCellActions: number;
+  disabledActions: string[];
 }
 
 export const InlineActions: React.FC<InlineActionsProps> = ({
   actionContext,
   showActionTooltips,
   visibleCellActions,
+  disabledActions,
 }) => {
   const { value: allActions } = useLoadActions(actionContext);
-  const { extraActions, visibleActions } = usePartitionActions(
-    allActions ?? [],
-    visibleCellActions
+  const filteredActions = useMemo(
+    () => (allActions ?? []).filter(({ id }) => !disabledActions?.includes(id)),
+    [allActions, disabledActions]
```

Comment:
This filtering code is duplicated, could we move this logic to the `useLoadActions` hooks? We could have a parameter `options?: { disabledActions?: string[] }`, so we can do:

`useLoadActions(actionContext, { disabledActions });`
or
`useLoadActionsFn({ disabledActions })`

This way we'll be able to add more things like `extraActions` to the _options_ parameter if needed.